apollo-compiler/src/main/kotlin/com/apollographql/apollo/compiler/OperationIdGenerator.kt

```diff
@@ -10,10 +10,9 @@ interface OperationIdGenerator {
        */
       operationDocument: String,
 
-      /**
-       * The path to the GraphQL file
-       */
-      operationFilepath: String
+      operationName: String,
+      operationPackageName: String
```

Comment:
That's a breaking change. In order to make the task cacheable, I had to remove `filePath` from the generated files which means that it's better if identification is made based on name and packageName. If needed, we could add `file.name` (without path) but I don't think this would help much compared to qualified name.

Reply:
I reverted that since it's not serialized anymore.